docs/operator-guide/upgrade/upgrade-edp-3.10.1.md

```diff
@@ -0,0 +1,251 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Upgrade KubeRocketCI v3.9 to 3.10.1
```

Comment:
lets use 3.10 instead of 3.10.2. Maybe we can use subsection for future releases